collector/xml-eater.py

```diff
@@ -0,0 +1,48 @@
+from lxml import etree 
+import numpy as np
+import re
+#xml_file: file to be parsed
```

Comment:
Can these comments be converted into a docstring instead? Something like this:

```python
def xml_eater(...):
    """
    Some description of what the function does.

    xml_file: ...
    sample_rate: ...
    """

Reply:
Yes, those should all be docstrings, I will convert them later.

Reply:
Fixed.